Community Balance Patch/Balance Changes/Text/en_US/PolicyText.sql

```diff
@@ -961,11 +961,11 @@ WHERE Tag = 'TXT_KEY_POLICY_FREERELIGION_TEXT' AND EXISTS (SELECT * FROM COMMUNI
 
 -- Monastery
 UPDATE Language_en_US
-SET Text = '+3 [ICON_FOOD] Food, +3 [ICON_RESEARCH] Science, +1 [ICON_PEACE] Faith.'
+SET Text = 'Religious building purchased with [ICON_PEACE] Faith. Increases City [ICON_FOOD] Food, [ICON_RESEARCH] Science and [ICON_PEACE] Faith output. Requires the adoption of the Fealty Policy Branch.'
```

Comment:
Why did you remove the quantity of each bonus?

Reply:
https://forums.civfanatics.com/threads/text-updates.647903/#post-15498779